extensions/Worker.Extensions.Storage.Tables/src/Worker.Extensions.Storage.Tables.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</AssemblyName>
+    <RootNamespace>Microsoft.Azure.Functions.Worker.Extensions.Storage.Tables</RootNamespace>
+    <Description>Azure Table Storage extensions for .NET isolated functions</Description>
+    
+    <!--Version information-->
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
Might be a good idea to publish this as a preview package (1.0.0-preview1), validate everything with that version and then remove the preview suffix and publish the non-preview version.